### DISCUSSION TITLE
Make a tui with trogon

### DISCUSSION THREAD

**Discussion by peteris-zealid** [MatthiasValvekens/pyHanko#352] [Ideas]
I found this really cool project https://github.com/Textualize/trogon

Tried it and it works like advertised with just two lines of code you get an interactive swagger-like API documentation in the terminal. It does not have autocomplete for filenames which is sad, but maybe the tool is still worth having.

**Comment by MatthiasValvekens** [comment]
Looks pretty cool! I'll take a look when I have the time.
